test/build_test.go

```diff
@@ -146,3 +148,40 @@ RUN cp /etc/foo /etc/bar
 	wg.Wait()
 	b.StopTimer()
 }
+
+func benchmarkBuildFileTransfer(b *testing.B, sb testutil.Sandbox) {
+	var appliers []fstest.Applier
+	appliers = append(appliers,
+		fstest.CreateDir("subdir1", 0755),
+		fstest.CreateFile("subdir1/file1.txt", []byte("foo"), 0600),
+		fstest.CreateFile("subdir1/file2.txt", make([]byte, 1024*1024), 0600), // 1MB file
+		fstest.CreateDir("subdir1/subdir2", 0755),
+		fstest.CreateFile("subdir1/subdir2/file3.txt", []byte("bar"), 0600),
+		fstest.CreateFile("subdir1/subdir2/file4.txt", make([]byte, 1024*1024*10), 0600), // 10MB file
+	)
+	for i := 0; i < 5000; i++ {
+		appliers = append(appliers, fstest.CreateFile(fmt.Sprintf("subdir1/file%d.txt", i+5), []byte("Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed do eiusmod tempor incididunt ut labore et dolore magna aliqua."), 0600))
+	}
+	appliers = append(appliers,
+		fstest.CreateFile("subdir1/largefile1.txt", make([]byte, 1024*1024*50), 0600),  // 50MB file
+		fstest.CreateFile("subdir1/largefile2.txt", make([]byte, 1024*1024*100), 0600), // 100MB file
+	)
+
+	dockerfile := []byte(`
+FROM busybox:latest
```

Comment:
Is this `busybox` cached or pulled (or pulled from local registry). If pulled from remote then better to use local image to not skew results.

Reply:
Yes it's cached on local registry